karaf/itest/src/test/java/org/apache/brooklyn/KarafTestUtils.java

```diff
@@ -44,7 +46,9 @@
         configureConsole().ignoreLocalConsole(),
         logLevel(LogLevel.INFO),
         features(karafStandardFeaturesRepository(), "eventadmin"),
-        junitBundles()
+        junitBundles(),
+        editConfigurationFilePut("etc/org.apache.brooklyn.osgilauncher.cfg", "persistMode", "AUTO"),
+        editConfigurationFilePut("etc/org.apache.brooklyn.osgilauncher.cfg", "persistenceDir", new File("target/paxexam/persistence/" + Identifiers.makeRandomId(6) + "/").getAbsolutePath()),
```

Comment:
That was my thinking, yes. Isn't worth making it more complex just to do the cleanup.